homeassistant/components/nzbget/config_flow.py

```diff
@@ -94,13 +95,9 @@ async def async_step_user(
             vol.Optional(CONF_PASSWORD): str,
             vol.Optional(CONF_PORT, default=DEFAULT_PORT): int,
             vol.Optional(CONF_SSL, default=DEFAULT_SSL): bool,
+            vol.Optional(CONF_VERIFY_SSL, default=DEFAULT_VERIFY_SSL): bool,
         }
 
-        if self.show_advanced_options:
```

Comment:
Why do we remove this?

Reply:
SSL options have been promoted to the main configuration window as the verification of SSL is needed in many reverse proxy situations. Hiding this one and only option behind another click can add confusion for users utilising this config flow

Reply:
If you turn on SLL but need to turn off SSL verification, you should know what you're doing and be an advanced user.